test/test_xpu.py

```diff
@@ -46,6 +53,33 @@ def test_get_device_properties(self):
         self.assertTrue(device_capability["max_work_group_size"] > 0)
         self.assertTrue(device_capability["max_num_sub_groups"] > 0)
 
+    @unittest.skipIf(
+        TEST_WITH_TSAN,
+        "TSAN is not fork-safe since we're forking in a multi-threaded environment",
+    )
+    @unittest.skipIf(IS_WINDOWS, "not applicable to Windows (only fails with fork)")
```

Comment:
You are not testing it with TSAN nor on Windows, why add those guards?

Reply:
Yes, you are right. Removed.